app/views/clinics/_card.html.erb

```diff
@@ -12,6 +14,7 @@
         <% end %>
```

Comment:
Would be helpful if we added a comment here .... as a TO DO. Add review after we create the review model